hibernate-core/src/main/java/org/hibernate/boot/model/process/internal/VersionResolution.java

```diff
@@ -22,6 +22,8 @@
 import org.hibernate.type.descriptor.java.MutabilityPlan;
 import org.hibernate.type.descriptor.jdbc.JdbcType;
 import org.hibernate.type.descriptor.jdbc.JdbcTypeIndicators;
+import org.hibernate.type.internal.UserTypeJavaTypeWrapper;
+import org.hibernate.type.internal.UserTypeVersionJavaTypeWrapper;
```

Comment:
Are the above two imports really used in this class? Seems not from your code changes below.